sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java

```diff
@@ -910,7 +1103,8 @@ public PagedIterable<QueueMessageItem> receiveMessages(Integer maxMessages, Dura
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public PeekedMessageItem peekMessage() {
-        return client.peekMessage().block();
+        List<PeekedMessageItem> result = peekMessages(null, null, null).stream().collect(Collectors.toList());
```

Comment:
Should `maxMessages` be set to `1` here? If this is set to `null` will the service just return a single message?